### DISCUSSION TITLE
Making fragmented lists half visible like Beamer overlays

### DISCUSSION THREAD

**Discussion by mil-ad** [marp-team/marp#146] [Q&A] [answered]
I was wondering if it's possible to tweak my CSS to replicate what Beamer overlays do with fragmented list, i.e. make upcoming items are half visible:

![image](https://user-images.githubusercontent.com/686449/129871463-75a84a32-48b2-44d0-ba78-f08fe6a8d2a5.png)

I _think_ marpit doesn't actually generate multiple `<section>`s for each fragment and instead adds some magic attributes to list items that are handled by the higher level app. Is there a way to achieve this through CSS customisation If I only care about the vanilla HTML output?

**Comment by yhatt** [comment, marked as answer]
> 🚚  Transferred from [marp-team/marpit#309](https://github.com/marp-team/marpit/issues/309) for discoverability by others.

Because Marpit framework itself does have no opinions about how to control fragmented list, styling is also depending on the higher level app.

Many Marp users are using Marp CLI's bespoke template, which is a practical refference implementation of HTML. In this case, you can style hidden items by below snippet:

```html
<style>
  [data-bespoke-marp-fragment="inactive"] {
    visibility: visible !important;
    opacity: 0.25;
  }
</style>
```

**Reply by mil-ad** [reply]
Thanks! Exactly what I was looking for
